core/tests/ui/desktop/e2e/checkout.spec.ts

```diff
@@ -3,14 +3,19 @@ import { expect, Page, test } from '@playwright/test';
 
 const sampleProduct = '[Sample] Laundry Detergent';
 const testUser = faker.person.firstName();
+const actionTimeout = 10000;
 
 async function enterShopperDetails(page: Page) {
-  await page.getByLabel('First Name').fill(testUser);
-  await page.getByLabel('Last Name').fill(faker.person.lastName());
-  await page.getByLabel('Company Name (Optional)').fill('BigCommerce');
-  await page.getByLabel('Phone Number (Optional)').fill(faker.phone.number());
-  await page.getByLabel('Address', { exact: true }).fill(faker.location.buildingNumber());
-  await page.getByLabel('City').fill('Natick');
+  await page.getByLabel('First Name').fill(testUser, { timeout: actionTimeout });
```

Comment:
This test has been failing a few times unexpectedly due to either the targeted element is not filled with desired text or the focus moves too quick resulting in mistyped string or partial fill. The only alternative to `fill`, `type` has been deprecated so, I opted to use `timeout` on these specific actions. Also added, `retry` for this one test since it's determined to be flaky until we have a robust fix for this.